pkgs/desktops/deepin/apps/deepin-image-viewer/default.nix

```diff
@@ -35,6 +35,11 @@ stdenv.mkDerivation rec {
       url = "https://github.com/linuxdeepin/deepin-image-viewer/commit/4a046e6207fea306e592fddc33c1285cf719a63d.patch";
       sha256 = "sha256-aIgYmq6WDfCE+ZcD0GshxM+QmBWZGjh9MzZcTMrhBJ0=";
     })
+    (fetchpatch {
+      name = "fix build with libraw 0.21";
+      url = "https://raw.githubusercontent.com/archlinux/svntogit-community/packages/deepin-image-viewer/trunk/libraw-0.21.patch";
```

Comment:
This url is not permanent, please point it to a specific commit.

Reply:
Commit has PKGBUILD changes，Changed to directly submit this patch file

Reply:
https://raw.githubusercontent.com/archlinux/svntogit-community/2ff11979704dd7156a7e7c3bae9b30f08894063d/trunk/libraw-0.21.patch